src/ngMessages/messages.js

```diff
@@ -44,25 +50,34 @@ var jqLite = angular.element;
  * </form>
  * ```
  *
- * Now whatever key/value entries are present within the provided object (in this case `$error`) then
- * the ngMessages directive will render the inner first ngMessage directive (depending if the key values
- * match the attribute value present on each ngMessage directive). In other words, if your errors
- * object contains the following data:
+ * In order to show error messages corresponding to `myField` we first create a container element with an `ngMessages` attribute
+ * set to the `$error` object owned by the `myField` input in our `myForm` form.
+ *
+ * Within this container we create separate elements for each of the possible errors that `myField` could be annotated with. Exactly
```

Comment:
`annotated` is a poor description. It's simply "an element for each error myField could have"